wrapper/src/test/java/software/amazon/jdbc/plugin/AwsSecretsManagerConnectionPluginTest.java

```diff
@@ -356,10 +357,36 @@ public void testConnectWithWrappedPostgreSQLException() throws SQLException {
     assertEquals(TEST_PASSWORD, TEST_PROPS.get(PropertyDefinition.PASSWORD.name));
   }
 
+  @ParameterizedTest
+  @MethodSource("arnArguments")
+  public void testConnectViaARN(final String arn, final Region region) throws SQLException {
```

Comment:
I think we need another test to check optional/required parameters (secretId and region)